packages/desktop-client/src/components/Notifications.tsx

```diff
@@ -172,13 +177,7 @@ function Notification({
                 color: 'currentColor',
                 fontSize: 14,
                 flexShrink: 0,
-                '&:hover, &:active': {
-                  backgroundColor: positive
-                    ? colors.g9
-                    : error
-                    ? colors.r10
-                    : colors.y9,
-                },
+                ...buttonStyle,
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏is this change necessary? Functionally it's exactly the same

Reply:
I had to do these because React's  `CSSProperties` does not define these and results in a compiler error. This is a way to get around that. Do you have any suggestions how else we can address this? Does making our own `CSSProperties` which extends React's and add a `[propertyName: string]: unknown` property make more sense?

Reply:
Yes, I think a custom type would be more sensible at this point. Reason being: we're still internally using `glamor` and not fully native React `style`. And we won't be able to port over to native until we solve all these issues. It will be easier to spot and solve them if a custom type is used rather than manual type coercions and workarounds.